doc/main.dox

```diff
@@ -250,13 +250,13 @@ notation. Here is an example:
 you lose all configuration overrides for both the old and new task. This limitation is not
 shared by \ref lsst.pex.config.registry.RegistryField "lsst.pex.config.RegistryField".
 
-To retarget a subtask is specified as an
+To retarget a subtask specified as an
 \ref lsst.pex.config.registry.RegistryField "lsst.pex.config.RegistryField"
 set the `name` property of the field. Overriding config parameters is a bit more complicated
 than for \ref lsst.pex.config.configurableField.ConfigurableField "lsst.pex.config.ConfigurableField":
 you can set the `active` attribute of the field to override the currently active subtask,
```

Comment:
Should this be something like "you can use the \`active\` attribute to override parameters of the currently active subtask"?